tests/test_observable.py

```diff
@@ -18,49 +18,24 @@
 from pennylane import numpy as np
 from scipy.linalg import block_diag
 
-from defaults import pennylane as qml
-from pennylane.expval import Identity
+import pennylane as qml
 from pennylane.qnode import QuantumFunctionError
 from pennylane.plugins import DefaultQubit
 
 import pytest
 
 
-def test_identity_raises_exception_if_outside_qnode():
```

Comment:
how come these were removed?

Reply:
Previously, rather than having a single `Identity` in `ops/__init__.py`, we were doing a bit of metaprogramming (similar to the `ExpvalFactory()`) in order to dynamically load the CV/qubit identity operation. This wasn't a very nice solution in practice though; the logic was complicated and likely had lots of edge cases, and it would not behave like a standard `Operation`.

Rather than spend time fixing it up to work with the new `expval()` function, I simply replaced `PlaceholderIdentity()` with `Identity(CVObservable, Observable)`, which appears to work identically, minus the metaprogramming/complicated logic.